docs/source/install.rst

```diff
@@ -19,9 +19,9 @@ If you're new to Python, we recommend installing `Anaconda <https://www.anaconda
 a Python distribution which includes nbconvert and the other Jupyter components.
 
 .. important::
-    
-    To unlock nbconvert's full capabilities requires Pandoc and TeX 
-    (specifically, XeLaTeX). These must be installed separately.
+
+    To unlock its full capabilities, nbconvert requires Pandoc, TeX
+    (specifically, XeLaTeX) and Chromium. These must be installed separately.
```

Comment:
Not a blocker, but I feel like this could be made more binary, and a bit less dramatic:

```
Exporting to PDF files requires additional, separately installed dependencies.
```

Reply:
Also, is pandoc required for this PR's workflow?

Reply:
Pandoc is used extensively by a range of preprocessors and exporters, so yes it's required in the general sense but not for the webpdf installation specifically. I can try rewording this to match xelatex vs web pdf conversion requirements.